xml_inscriptions/transcribed/NY.NY.MMA.G.74.51.2266.xml

```diff
@@ -0,0 +1,177 @@
+                            <p>The <persName role="collector"><name key="Cesnola">Cesnola</name></persName> Collection, Purchased by subscription, <date notBefore="1873" notAfter="1877">1874–76</date>.</p>
+                        </provenance>
+                        
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Myres"/>
+                        <biblScope unit="item">1836</biblScope>
+                    </bibl>
+                    <bibl> 
+                        <ref type="external" target="https://www.metmuseum.org/art/collection/search/241819">Metropolitan Museum Web Entry</ref> 
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+        
+        <encodingDesc>
+            
+            <xi:include
+                href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                <xi:fallback>
+                    <p>Taxonomies for US Epigraphy controlled values</p>
+                </xi:fallback>
+            </xi:include>
+            
+        </encodingDesc>
+        
+        <profileDesc> <!-- not used --> </profileDesc>
+        
+        
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+        <!-- Record all changes here -->
+        
+        <revisionDesc>
+            <change when="2021-04-15" who="Itamar Levin">Transcription , translation, and metadata</change>
+        </revisionDesc>
+        
+    </teiHeader>
+    
+    <!--  *************************************  <facsimile>  *********************************  -->
+    <!-- Used to record images of an inscription.   -->
+    
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="NY.NY.MMA.G.74.51.2266.jpg"/>
+        </surface>
+    </facsimile>
+    
+    <text>
+        <body>
+            <div type="edition" subtype="transliteration_ltn">
+                <ab>
+                    <lb n="1" style="text-direction:l-to-r"/><supplied reason="lost">ta. se.</supplied> pa. pi. a. se.
+                </ab></div>
+            <div type="edition" subtype="transliteration_grk">
+                <ab>
+                    <lb n="1" style="text-direction:l-to-r"/><supplied reason="lost">τᾶς</supplied> <name type="divine" key="Παφία">Παφίας</name>.
+                </ab></div>
+            
+            <div type="translation">
+                <p>Of Paphia.</p></div>
+            
+            <div type="commentary">
+                <p>Transcription from Myres (1914). Written in Cypriot syllabary.</p></div>
```

Comment:
In most commentary notes I've removed the "read from right to left" after the Cypriot syllabary comment; since that is the default for Cypriot syllabary, I'd argue that the exceptional instances where there are deviations are what we should be noting, rather than the reading direction that is inherent in the script.